packages/graphics/src/Graphics.ts

```diff
@@ -1217,12 +1219,8 @@ export class Graphics extends Container
                 const g = (tintRGB[1] * batchTint[1]) * 255;
                 const b = (tintRGB[2] * batchTint[2]) * 255;
 
-                // TODO Ivan, can this be done in one go?
-                const color = (r << 16) + (g << 8) + (b | 0);
-
-                batch._tintRGB = (color >> 16)
-                        + (color & 0xff00)
-                        + ((color & 0xff) << 16);
+                // assemble word-order RGB
+                batch._tintRGB = (b << 16) + (g << 8) + (r | 0);
```

Comment:
Why `r | 0`, that doesn't change anything?

Reply:
Is there a bug for this change @ivanpopelyshev? Need some verification here.

Reply:
There's no bug.
However, now that I think about it, `round` will work better there.